src/app/Domains/Challenge/UseCases/UpdateChallengeUseCase.php

```diff
@@ -56,7 +56,7 @@ public function __invoke(int $id, ChallengeUpdateRequest $request): MysChallenge
                 // TODO: arrayで文字列指定になってるのが気に入らない。
                 $detail = MysChallenge2ChallengeDetail::find($requestDetail['id']);
                 // detailで自分が親じゃないものを指定しているときは更新しない
-                if ($detail->challenge->id == $id) {
+                if (isset($detail) && $detail->challenge->id == $id) {
```

Comment:
```suggestion
                if ($detail?->challenge?->id == $id) {
```

一応こういうのでもいけますけど、全体的にGoっぽい書き方で統一してる方がいいか

Reply:
PHPならPHPの書き方にあわせるほうが良いと思いますねー
ぶたまんさんの書き方のほうがスマートだしどこかでその書き方に統一しましょうか。